discourse/discourse-update.dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM janx/servo
+MAINTAINER Jan Keromnes "janx@linux.com"
+MAINTAINER Michael Howell "michael@notriddle.com"
+
+# Upgrade all packages.
+RUN sudo apt-get update -q && sudo apt-get upgrade -qy && rustup update
```

Comment:
Does the Discourse project really use Rust? I'm not really against updating it, but I found this command surprising.